src/hardware/developing-an-iox/protocol.md

```diff
@@ -22,27 +30,30 @@ The Arbitration ID Field for IOX Messages:
 
 | Bits | 28 to 22 | 21 to 16 | 15 to 0 |
 | --- | --- | --- | --- |
-| Contents | Reserved: 0 | Message: 0–63 | All IOXs: 0 <br> Individual IOX ID: 1–65535 |
+| Contents | Reserved: 0 | Message: 0–63 | All IOXs: 0 <br> Individual IOX Destination ID: 1–65535 |
 
 0x1FC00000 IO\_EXPANDER\_RESERVED\_MASK
 
 0x003F0000 IO\_EXPANDER\_COMMAND\_MASK
 
 0x0000FFFF IO\_EXPANDER\_ID\_MASK
 
-### Identifier
+### Destination ID
```

Comment:
As you noted, this ID functions as both a destination (when being sent from the GO) or a source (when being sent from an IOX). As such, I think the naming "Destination ID" would likely cause confusion.

Reply:
How about changing that to "Address ID"? @jwidz feel free to chime in.

Reply:
Address ID sounds fine to me.

Reply:
@muyuncai test

Reply:
I can comment

Reply:
Jason couldn't comment, but he's fine with "Address ID" too